Bowling.java

```diff
@@ -87,9 +92,8 @@ public void update(Integer pinsDown) {
 		Frame currentFrame = getCurrentFrame();
 		currentFrame.update(pinsDown);
 
-		if (currentFrame.isStrike()) {
+		if (currentFrame.isStrike())
```

Comment:
Also, what does calling `currentFrame.update(0)` do here?  If it's important I'm surprised we don't need to do it during a spare as well?